### DISCUSSION TITLE
日期无法显示

### DISCUSSION THREAD

**Discussion by muriatec** [sjtug/SJTUThesis#781] [提问] [answered]
对setup.tex里的date无论进行何种修改，封面页都不会显示日期

**Comment by LogCreative** [comment, marked as answer]
本科论文封面是没有日期的，可能你想问的是课程论文的封面，如果坚持想要添加日期的话，可以在

https://github.com/sjtug/SJTUThesis/blob/3436a7b82eb6d63f3085cb7545b84baf826e03e5/texmf/tex/latex/sjtuthesis/sjtuthesis-undergraduate.ltx#L153

**之后**添加类似于硕士/博士论文的这行

https://github.com/sjtug/SJTUThesis/blob/3436a7b82eb6d63f3085cb7545b84baf826e03e5/texmf/tex/latex/sjtuthesis/sjtuthesis-graduate.ltx#L137-L138

但是由于本科子类没有定义日期名称，所以应当添加的是这样一行：
```latex
        日期       &
          \sjtu@date{\sjtu@date@format@zh}{\sjtu@info@date}           \\\cline{2-2}
```
